test/job_hooks_test.rb

```diff
@@ -342,6 +342,7 @@ def self.perform(history)
     Resque.dequeue(BeforeDequeueJob, history)
     @worker.work(0)
     assert_equal history, [:before_dequeue], "before_dequeue was not run"
+    assert_equal [:before_dequeue], history, "before_dequeue was not run"
```

Comment:
I was just wondering why both asserts are needed. Will `assert_equal history, [:before_dequeue], "before_dequeue was not run"` ever not raise when `assert_equal [:before_dequeue], history, "before_dequeue was not run"` would raise?